modules/core/src/main/java/org/apache/ignite/internal/hlc/ClockService.java

```diff
@@ -31,13 +31,29 @@ public interface ClockService {
      */
     HybridTimestamp now();
 
+    /**
+     * Gets a current timestamp. It is a fast way to get timestamp because it does not have to update internal state.
```

Comment:
I'd rather say "doesn't have to tick the logical part of the clock". Within the context of hybric clock that should be more clear. Here and in other comments.